File: offers/back/Dbo/Companies/Company.cs
using System;
using System.Collections.Generic;

namespace AiOffer.Dbo.Companies
{
    public partial class Company
    {
        public ulong Id { get; set; }
        public string Mail { get; set; }
        public string HashedPassword { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Description { get; set; }
    }
}

File: offers/front/src/app/models/company.ts
import { ApiCompany } from './api/api-company';

export class Company {
    public readonly id: bigint;
    public readonly mail: string;
    public readonly name: string;
    public readonly address: string;
    public readonly description: string;

    constructor(json: ApiCompany) {
        this.id = json.id;
        this.mail = json.mail;
        this.name = json.name;
        this.address = json.address;
        this.description = json.description;
    }
}